ra2ce/runners/direct_analysis_runner.py

```diff
@@ -58,65 +61,124 @@ def can_run(ra2ce_input: ConfigWrapper) -> bool:
         return True
 
     @staticmethod
-    def _get_result_link_based(analysis: AnalysisDirectProtocol, analysis_config: AnalysisConfigWrapper,
-                               result_segment_based: gpd.GeoDataFrame) -> gpd.GeoDataFrame:
-        # Step 00: define parameters
+    def _calculate_link_damage(
```

Comment:
Why not create this as an inner function of `_get_result_link_based` just like `get_damage_result_columns`? It's private and not used elsewhere.